test/nodes/test_generator.py

```diff
@@ -64,8 +63,8 @@ def test_generator_pipeline(document_store, retriever, rag_generator, docs_with_
 def test_lfqa_pipeline(document_store, retriever, lfqa_generator, docs_with_true_emb):
     # reuse existing DOCS but regenerate embeddings with retribert
     docs: List[Document] = []
-    for idx, d in enumerate(docs_with_true_emb):
-        docs.append(Document(d.content, str(idx)))
+    for _, d in enumerate(docs_with_true_emb):
```

Comment:
You can simplify this further:
```suggestion
    for d in docs_with_true_emb:
```

Reply:
Yes, I thought about it, then I looked around, and I couldn't find any reason for the idx being applied. I thought I were missing something and just let it there. :smile: 

I will remove it.